### DISCUSSION TITLE
extracting frontend code issue

### DISCUSSION THREAD

**Discussion by aambadkar** [learndevopsonline/learndevopsonline#1444] [Q&A] [answered]
hi,
in frontend extracting code from file is going to failed. even same function is working for backend

![error](https://github.com/learndevopsonline/learndevopsonline/assets/143367591/fcde94a8-ebdf-4bf3-bf60-c55e354f4544)

**Comment by sreedharm07** [comment, marked as answer]
Hi @aambadkar ,
before extracting code download frontend code . you missed the step to download frontend code

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/0d7e3e6f-f34b-480f-b128-b047a16b0e6d)
